src/scss/main.scss

```diff
@@ -0,0 +1,107 @@
+@import '~bootstrap/scss/bootstrap';
+// Variables
+$purple-color: #a64fae;
+$white-color: #fff;
+
+body {
+  font-family: 'poppins', sans-serif;
+  background-color: grey;
+  background-color: $white-color;
+}
+
+.btn-flat {
+  background-color: #5d5;
+  color: #000;
+}
+
+.colcenter {
+  padding: 1rem;
+  text-align: center;
+}
+
+button.btn-form {
+  background-color: $purple-color;
+  color: $white-color;
+  border: 0;
+  font-family: 'poppins', sans-serif;
+  margin-top: 25px;
+  border-radius: 5px;
+  height: 40px;
+}
+
+.btn-upload {
+  background-color: $purple-color;
+  color: $white-color;
+  border: none;
+  font-family: 'poppins', sans-serif;
+  margin: 15px 145px;
+  border-radius: 5px;
+  height: 45px;
+  width: 100px;
+}
+
+.tag-btn {
+  font-family: 'poppins', sans-serif;
+  border: 2px solid #ebeaed;
+  color: $purple-color;
+}
+
+.tag-btn:hover {
+  background-color: $purple-color;
+  border-color: $white-color;
+}
+
+h2 {
+  line-height: 1.66;
+  margin: 0;
+  padding: 0;
+  font-weight: bold;
+  margin-bottom: 10px;
+  color: $purple-color;
+  font-family: 'poppins', sans-serif;
+  font-size: 36px;
+}
+
+.text-field {
+  width: auto;
+  height: 3rem;
+  border-radius: 10px;
+  border: solid 1px #000;
+}
+
+.form-group-set {
+  margin-top: 1.8rem;
+}
+
+/*   Company Match Page Styling */
+.view-btn {
+  background-color: $purple-color;
+  border: none;
+  margin-right: 6px;
+  height: 30px;
+  align-items: center;
+}
+
+.match-card {
+  border: solid 1px;
+  border-radius: 5px;
+}
+
+.nav-pills .nav-link {
+  font-size: semi-bold;
+  font-weight: 2rem;
+  color: #000;
+}
+
+.nav-pills .nav-link.active {
+  color: $purple-color;
+  background-color: $white-color;
+}
+
+.side-nav-link {
+  margin: 10px;
+}
+
+Tab {
+  color: #000;
```

Comment:
$black

Reply:
done

Reply:
You still have the same 8 issues. Go back to the previous review I gave you and solve the issues, if you want your pr to be merged. I won't be giving you any more reviews on this pr, because my requests on the previous one are clear.